examples/misc_instancing.html

```diff
@@ -0,0 +1,231 @@
+        </script>
+
+        <script type="text/javascript">
+            // Define initial camera position
+            var positionOnGlobe = {
+                longitude: -0.57918,
+                latitude: 44.837789,
+                altitude: 3000,
+            };
+            var placement = {
+                coord: new itowns.Coordinates("EPSG:4326", -0.57918, 44.837789),
+                range: 1000,
+                tilt: 45,
+            };
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById("viewerDiv");
+
+            // Instanciate iTowns GlobeView*
+            var view = new itowns.GlobeView(viewerDiv, placement);
+
+            // Setup loading screen and debug menu
+            setupLoadingScreen(viewerDiv, view);
+            const debugMenu = new GuiTools("menuDiv", view);
+
+            var ambLight = new itowns.THREE.AmbientLight(0xffffff, 0.2);
+            view.scene.add(ambLight);
+
+            // Add one imagery layer to the scene
+            itowns.Fetcher.json("./layers/JSONLayers/Ortho.json").then(
+                function _(config) {
+                    config.source = new itowns.WMTSSource(config.source);
+                    var layer = new itowns.ColorLayer("Ortho", config);
+                    view.addLayer(layer).then(
+                        debugMenu.addLayerGUI.bind(debugMenu)
+                    );
+                }
+            );
+
+            //Tree
+            const trunkRadius = 5;
+            const trunkHeight = 20;
+            const topHeight = 10;
+
+            function makeTree() {
+                const root = new THREE.Object3D();
+
+                // Trunk
+                const geometry = new THREE.CylinderGeometry(
+                    trunkRadius,
+                    trunkRadius,
+                    trunkHeight,
+                    32
+                );
+                const material = new THREE.MeshPhongMaterial({
+                    color: 0x8b4513,
+                });
+                const trunk = new THREE.Mesh(geometry, material);
+                trunk.rotateX(Math.PI / 2);
+                trunk.position.z = 10;
+                trunk.updateMatrix();
+                root.add(trunk);
+
+                // Canopy
+                const geometryCanop = new THREE.SphereGeometry(
+                    topHeight,
+                    topHeight,
+                    10
+                );
+                const materialCanop = new THREE.MeshPhongMaterial({
+                    color: 0x00aa00,
+                });
+                const top = new THREE.Mesh(geometryCanop, materialCanop);
+                top.position.z = trunkHeight - topHeight / 3 + 10;
+                top.updateMatrix();
+                root.add(top);
+
+                return root;
+            }
+
+            // Load a gltf model
+            // Instantiate a loader
+            const loader = itowns.glTFLoader;
```

Comment:
Could you move this on line 157 ? Moreover, as it is only used once we may not need to create a `const` for that.